### PR TITLE
Updated LIBXSMM/DNN and revised TPP-benchmark plot

### PR DESCRIPTION
**Pull request by hfp**

### CODE REVIEW

File: scripts/buildkite/tpp-benchmark.yml [hunk 1]
```diff
@@ -19,6 +19,8 @@ steps:
               $${BUILDKITE_BUILD_CHECKOUT_PATH}/scripts/buildkite/build_tpp.sh'"
     env:
       LOGRPTSUM: "mlir"
-      #LOGRPTFMT: "svg"
+      LOGRPTFMT: "svg"
+      LOGRPTFMT: "svg"
```
Comment (rengolin) [inline scripts/buildkite/tpp-benchmark.yml:23]: Twice?

Comment (hfp) [reply scripts/buildkite/tpp-benchmark.yml:23]: Ohh, I did not save in the editor... let me update.

**Review by rengolin — APPROVED** [review]
Should be fine to merge whenever you fix the report. Thanks!

**Comment by hfp** [comment]
Buildkite applies CSS to an img component of the log with maximum height hard-coded to 500px.

**Comment by hfp** [comment]
There is no way to show results/report with more than 500px in height (without feature request to Buildkite). I will add multi-format support for the reporting tool like PNG/SVG and PDF, upload PDF with higher height, and only show PNG/SVG limited to 500px in height.

**Comment by hfp** [comment]
Updated PR will have URL embedded into log navigating to PDF-report with best details/readability.

**Comment by hfp** [comment]
> Updated PR will have URL embedded into log navigating to PDF-report with best details/readability.

This is reflected by the updated PR. However, we will also need an updated recipe to upload artifacts. Uploading artifacts cannot be done as part of a running job because our Slurm jobs run on compute nodes without agent.

**Pull request merged**